forge-game/src/main/java/forge/game/trigger/TriggerHandler.java

```diff
@@ -342,12 +342,9 @@ private boolean runWaitingTrigger(final TriggerWaiting wt) {
             }
         }
 
-        // AP
-        checkStatics |= runNonStaticTriggersForPlayer(playerAP, wt, delayedTriggersWorkingCopy);
-
-        // NAPs
-        for (final Player nap : game.getNonactivePlayers()) {
-            checkStatics |= runNonStaticTriggersForPlayer(nap, wt, delayedTriggersWorkingCopy);
+        // the trigger will be ordered later in MagicStack
+        for (final Player player : game.getPlayers()) {
```

Comment:
if we agree that the order here is irrelevant, then should just get rid of looping too